hypothesis-python/tests/cover/test_conjecture_engine.py

```diff
@@ -1539,25 +1518,6 @@ def shrinker(data):
     assert shrinker.calls <= 60
 
 
-def test_zero_examples_does_not_try_to_adapt_across_different_sizes():
```

Comment:
I removed this test because I couldn't figure out a sensible way to adapt it and it seemed fragile and hyper-specific.